streaming/src/reliability_helper.h

```diff
@@ -0,0 +1,68 @@
+#ifndef RAY_STREAMING_RELIABILITY_HELPER_H
+#define RAY_STREAMING_RELIABILITY_HELPER_H
+#include "reliability/barrier_helper.h"
+#include "channel/channel.h"
+#include "util/config.h"
+#include "data_reader.h"
+#include "data_writer.h"
+
+namespace ray {
+namespace streaming {
+
+class ReliabilityHelper;
+class DataWriter;
+class DataReader;
+
+class ReliabilityHelperFactory {
+ public:
+  static std::shared_ptr<ReliabilityHelper> GenReliabilityHelper(
+      const StreamingConfig &config, StreamingBarrierHelper &barrier_helper,
+      DataWriter *writer, DataReader *reader);
+};
+
+class ReliabilityHelper {
```

Comment:
I think we do not need atleaseonce/exactly once helper.
There is no difference if we send empty message in all reliability.

Reply:
We need it because FilterMessage is different between exactly once and at least once.